src/libsystemd/sd-device/sd-device.c

```diff
@@ -2598,7 +2598,7 @@ _public_ int sd_device_trigger_with_uuid(
 _public_ int sd_device_open(sd_device *device, int flags) {
         _cleanup_close_ int fd = -EBADF, fd2 = -EBADF;
         const char *devname;
-        uint64_t q, diskseq = 0;
+        uint64_t q, diskseq;
```

Comment:
Why the initialization is dropped? This will cause use before initialized below `if (q != diskseq)`.